helm/alfresco-content-services/values.yaml

```diff
@@ -331,7 +331,8 @@ alfresco-search:
 alfresco-digital-workspace:
   enabled: true
   ingress:
-    path: /digital-workspace
+    path: /workspace
```

Comment:
Is it required to change any documentation regarding this?

Reply:
It was changed in the past for the previous release, we just forgot to update the path for the k8s tests. See https://github.com/Alfresco/acs-deployment/blob/master/docs/docker-compose-deployment.md